Setup/UpgradeData.php

```diff
@@ -251,10 +271,10 @@ public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface
 
             // Add attributes to the eav/attribute
             $eavSetup->addAttribute(
-                \Magento\Catalog\Model\Product::ENTITY,
+                Product::ENTITY,
                 'myparcel_classification',
                 [
```

Comment:
hier nog meer van die giga arrays...

Reply:
De paar arrays die nu nog over zijn waren iets te verschillend van de rest om ze ook op een nette manier in zo'n default array te krijgen